feature-libs/cart/base/components/clear-cart/clear-cart-dialog/clear-cart-dialog.component.ts

```diff
@@ -40,6 +39,7 @@ export class ClearCartDialogComponent implements OnDestroy {
   ) {}
 
   clear(): void {
+    this.isClearing = true;
```

Comment:
Do we rely on the modal being closed once the processing is over to make the spinner go away?

Reply:
Yes, there is no disabling it. Maybe we can discuss a better way? I did this because now I capture both success and fail and close the dialog upon receiving either event